CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - fix: provide nodejs_version_info metric value after calling `registry.resetMetrics()` (#238)
 - fix: provide process_max_fds metric value after calling `registry.resetMetrics()`
 - fix: provide process_start_time_seconds metric value after calling `registry.resetMetrics()`
+- fix: fix regression in 4e6aacd (unreleased) (#382)
```

Comment:
do we need to add this? since it's not released I think it might just cause confusion for people using the changelog to migrate?

Reply:
I agree. CI will fail without it but looks like we can merge despite that failure.